src/Injector.tsx

```diff
@@ -91,11 +91,12 @@ export const Injector: React.FC<React.PropsWithChildren<Props>> = ({
         environmentHandler,
         authHelper,
         archiveHelper,
-        keycloakController,
+        authController,
       }}
     >
       <UpdateBanner apiHelper={apiHelper} />
       {children}
+      <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
Why did you add it a second time? I believe we only need it on the root component, no?

Reply:
Forgot to remove that, after merging. removed